lib/elixir_analyzer.ex

```diff
@@ -49,25 +51,32 @@ defmodule ElixirAnalyzer do
   """
   @spec analyze_exercise(String.t(), String.t(), String.t(), keyword()) :: Submission.t()
   def analyze_exercise(exercise, input_path, output_path, opts \\ []) do
+    Logger.info("Analyzing #{exercise}")
+
+    Logger.debug("Input: #{input_path}, Output: #{output_path}")
+
     params = get_params(exercise, input_path, output_path, opts)
 
-    s =
+    submission =
```

Comment:
I see you lived to regret choosing single-letter variable names 😁

Reply:
I would like to think all of those functions started much smaller at one time, then they got bigger and I never made the change. 🤷🏻 In reality, it was probably just a bad choice at the time.